Sources/Rendering/Core/ImageMapper/index.d.ts

```diff
@@ -257,11 +257,28 @@ export interface vtkImageMapper extends vtkAbstractImageMapper {
 	 */
 	getSlicingModeNormal(): number[];
 
+	/**
+	 * Get the slicing mode. 
+	 */
+	getSlicingMode(): number;
```

Comment:
Can you use the `SlicingMode` from `Constants.d.ts`? if you change also for the setter that would be good.